src/lib/utils/functions/getDataFromFromStorage.ts

```diff
@@ -1,7 +1,12 @@
-import { EntityHeader, Evaluation, UserProfile } from '../synapseTypes'
+import {
+  ColumnType,
+  EntityHeader,
+  Evaluation,
+  UserProfile,
+} from '../synapseTypes'
 import { SynapseConstants } from '..'
 
-export const getStoredEntityHeaders = (): EntityHeader[] => {
+const getStoredEntityHeaders = (): EntityHeader[] => {
```

Comment:
Don't want to encourage using these, so don't export anything that we don't need to export.